docker-compose.override.yml

```diff
@@ -0,0 +1,41 @@
+---
+version: '3.1'
```

Comment:
Can you do the same with environment variables: https://docs.docker.com/compose/environment-variables/ and 2 small bash scripts `run-production-site.sh` and `run-development-shite.sh` you could achieve the same and it will be more readable.

Reply:
You mean to replace yml files with shell scripts to make it more readable?

Reply:
`docker-compose.yml` and `docker-compose.override.yml` are standard files recognized by docker compose.

`docker-compose.prod.yml` is a convention and is not really necessary here.

I could use just the `docker-compose.yml` and not create the other two yml files but it's a docker compose convention to have three separate ymls.